server/public/main.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
You shouldn't disable this in your projects, the rule is meant to keep your code clean.